src/components/Inputs/index.tsx

```diff
@@ -16,16 +16,17 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { motion, Variants } from 'framer-motion'
+import { HTMLMotionProps, motion, Variants } from 'framer-motion'
+import { FC } from 'react'
 import styled, { css } from 'styled-components'
 import tinycolor from 'tinycolor2'
 
 export interface InputProps extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'placeholder'> {
+  label?: React.ReactNode
```

Comment:
Since we already have the `react` import, why not import `ReactNode` as well?

This makes me think, since `React` is imported automagically (due to our typescript settings), we could avoid all `import ... from 'react'` lines in our code by just using the `React` object everywhere... 🤔

Reply:
Imported `ReactNode` and `InputHTMLAttributes `.
I find the code more readable when note using `React.` everywhere :)